CONTRIBUTING.md

```diff
@@ -51,6 +51,9 @@ Other items:
   to break other users' code, so care must be taken not to introduce bugs.
 * Be aware that the pull request review process is not immediate, and is
   generally proportional to the size of the pull request.
+* Function arguments:
+  - Use full names for parameters rather than symbols (e.g. temperature instead of t)
+  - Order: pressure/height -> temperature -> moisture (in general, but not a hard and fast rule)
```

Comment:
Would it make sense to add wind components or grid parameters somewhere to this order?

On a similar note, three functions have a `pressure, u, v, height, ...` signature (`bunkers_storm_motion`, `bulk_shear`, and `critical_angle`). Is this exception small enough to gloss over or big enough to mention here?

Reply:
I'll clarify that default arguments are a good reason in `CONTRIBUTORS.md`.

Reply:
Also, added wind.